docs/pre.sh

```diff
@@ -3,40 +3,45 @@
 ## Create modules pages
 for D in ../x/*; do
   if [ -d "${D}" ]; then
-    MODDOC=docs/modules/$(echo $D | awk -F/ '{print $NF}')
+    MODDOC=docs/integrate/modules/$(basename "$D")
     rm -rf $MODDOC
-    mkdir -p $MODDOC && cp -r $D/README.md "$_"
+    mkdir -p $MODDOC && cp -r $D/README.md "$MODDOC/$(basename "$D").md"
   fi
 done
 
 ## Vesting is a submodule of auth, but we still want to display it in docs
 ## TODO to be removed in https://github.com/cosmos/cosmos-sdk/issues/9958
-cp ../x/auth/vesting/README.md ./docs/modules/auth/1-vesting.md
-cp ../x/auth/tx/README.md ./docs/modules/auth/2-tx.md
+cp ../x/auth/vesting/README.md ./docs/integrate/modules/auth/1-vesting.md
+cp ../x/auth/tx/README.md ./docs/integrate/modules/auth/2-tx.md
 
 ## Add modules page list
-cat ../x/README.md | sed 's/\.\.\/docs\/building-modules\/README\.md/\/building-modules\/intro\.html/g' > ./docs/modules/README.md
+cat ../x/README.md | sed 's/\.\.\/docs\/integrate\/building-modules\/README\.md/\/building-modules\/intro\.html/g' > ./docs/integrate/modules/"$MODDOC/$(basename "$D").md"
+sed -i -e "s|(../core/|(../../develop/advanced-concepts/"./docs/integrate/building-modules/01-module-manager.md
 
 ## Add tooling documentation
-cp ../tools/cosmovisor/README.md ./docs/tooling/01-cosmovisor.md
-cp ../tools/confix/README.md ./docs/tooling/02-confix.md
-cp ../tools/hubl/README.md ./docs/tooling/03-hubl.md
+cp ../tools/proto/README.md ./docs/integrate/tooling/00-proto.md
+cp ../tools/cosmovisor/README.md ./docs/integrate/tooling/01-cosmovisor.md
+cp ../tools/confix/README.md ./docs/integrate/tooling/02-confix.md
+cp ../tools/hubl/README.md ./docs/integrate/tooling/03-hubl.md
 wget -O docs/run-node/04-rosetta.md https://raw.githubusercontent.com/cosmos/rosetta/main/README.md
 
 ## Add package documentation
-cp ../client/v2/README.md ./docs/core/17-autocli.md
-cp ../depinject/README.md ./docs/packages/01-depinject.md
-cp ../collections/README.md ./docs/packages/02-collections.md
-cp ../orm/README.md ./docs/packages/03-orm.md
+cp ../client/v2/README.md ./docs/develop/advanced-concepts/17-autocli.md
+cp ../depinject/README.md ./docs/integrate/packages/01-depinject.md
+cp ../collections/README.md ./docs/integrate/packages/02-collections.md
+cp ../orm/README.md ./docs/integrate/packages/03-orm.md
 
 ## Add architecture documentation
-cp -r ./architecture ./docs
+cp -r ./architecture ./docs/integrate
 
 ## Add spec documentation
-cp -r ./spec ./docs
+cp -r ./spec ./docs/integrate
 
 ## Add rfc documentation
-cp -r ./rfc ./docs
+cp -r ./rfc ./docs/integrate
 
 ## Add SDK migration documentation
-cp -r ../UPGRADING.md ./docs/migrations/02-upgrading.md
\ No newline at end of file
+cp -r ../UPGRADING.md ./docs/integrate/migrations/02-upgrading.md
+sed -i -e "s|./proto/README.md|../../../../proto/README.md|g" ./docs/integrate/migrations/02-upgrading.md
```

Comment:
What is this for?

Reply:
Because the file is copied over the existing link needs to remain the same whilst the copied file needs to contain the updated path otherwise the link wont work

Reply:
Yeah but proto/README.md is not included in the docs, so the links will still point to nowhere once it is generated right? Why not putting the direct GitHub link in the UPGRADING.md then?